packages/grafana-ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -16,6 +16,19 @@ export type PopoverContent = string | React.ReactElement<any> | ((props: Popover
 export const Tooltip: FC<TooltipProps> = React.memo(({ children, theme, ...controllerProps }: TooltipProps) => {
   const tooltipTriggerRef = createRef<HTMLElement | VirtualElement>();
   const popperBackgroundClassName = 'popper__background' + (theme ? ' popper__background--' + theme : '');
+  const closePopover = (event: React.KeyboardEvent<HTMLDivElement>, hidePopper: () => void) => {
```

Comment:
Noticed this piece of code is duplicated across the codebase. Would extract this into an internal util function and reuse. This would be done in a separate PR though